gslib/commands/ls.py

```diff
@@ -58,13 +58,11 @@
 
 <B>LISTING PROVIDERS, BUCKETS, SUBDIRECTORIES, AND OBJECTS</B>
   If you run gsutil ls without URLs, it lists all of the Google Cloud Storage
-  buckets under your default project ID:
+  buckets under your default project ID (or all of the {{gcs_name}} buckets
```

Comment:
Do we actually do text substitution in these? {{}}
I don't see other instances of this syntax or a "gcs_name" variable in the file

Reply:
No we don't! Good catch on this - I had copy-pasted from the c.g.c do, where these variables are relevant. Fixed.